packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/exception/Exceptions.kt

```diff
@@ -0,0 +1,41 @@
+package expo.modules.kotlin.exception
```

Comment:
It's probably fine as is, but on iOS I intentionally used `CommonExceptions.swift` filename since `Exception` already exists and I don't really like having mixed singular and plural forms 😅 It's much easier to jump into this file then.